decision_service/include/object_factory.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+#include "config_collection.h"
+#include "api_status.h"
+#include <functional>
```

Comment:
why in the middle? can we do include <..> include "..." afterwards?

Reply:
Changed

---
In reply to: [193420010](https://github.com/JohnLangford/vowpal_wabbit/pull/1495#discussion_r193420010) [](ancestors = 193420010)